### DISCUSSION TITLE
binary classification issue with proba not ranging between 0 and 1 but between 0.2 and 0.3

### DISCUSSION THREAD

**Discussion by fcoppey** [autogluon/autogluon#3104] [Q&A]
I’m trying to make a binary classifier with AutoGluon 0.7.0, no custom presets, just auto_stack=True, I have a real issue getting a prediction that ranges between 0 and 1 when I train the model for more than a few minutes... dataset has 7000 rows, full training takes 7h but then the prediction range of the binary data ranges in a much narrower range (the positive rows have a probability of 0.3 and the negative have 0.2 for instance. who can help me understand which setting is causing this narrow probability range?

![bokeh_plot-156](https://user-images.githubusercontent.com/42648441/229314251-8e10ff4a-8a78-48f0-86e3-8e11a7bffbd1.png)
![bokeh_plot-157](https://user-images.githubusercontent.com/42648441/229314253-6b028870-7285-4501-b77e-86ef24aa60e0.png)

when displaying the leaderboard of the model I can see individual models yielding very nice prediction and when I call the first model with predictor.predict_proba(dataset, model=‘first_model_from_leaderboard’) the proba ranges from 0 to 1 correctly. but I cannot figure out why it is not the case when I call predictor.predict_proba(dataset)
<img width="943" alt="Screenshot 2023-04-02 at 00 15 56" src="https://user-images.githubusercontent.com/42648441/229316474-ac9b6e2b-43df-45f4-b92b-b81ebf01c044.png">

**Comment by gradientsky** [comment]
As a workaround: have you considered re-scaling original labels to 0-1 range and then scaling back predictions to the expected range? I wonder if the issue remains if this is done.
